site/src/examples/slider/WithInput.tsx

```diff
@@ -0,0 +1,143 @@
+import {
+  FlexLayout,
+  FormField,
+  FormFieldLabel,
+  Input,
+  StackLayout,
+} from "@salt-ds/core";
+import {
+  Slider,
+  type SliderChangeHandler,
+  type SliderValue,
+} from "@salt-ds/lab";
+import { type ChangeEvent, type ReactElement, useState } from "react";
+
+export const SingleWithInput = () => {
```

Comment:
A little confused on the behaviour between input & slider.
If I input "2222" in input, it's "valid"?, then tab to slider, tooltip shows "2222" when i tab away, it falls back to "50"

Reply:
Changed the validation method for all stories and examples with inputs. Now behaves much more predictably. Invalid values trigger the input validation state, and only valid values are passed to the actual slider.